components/Layout.vue

```diff
@@ -1,91 +1,81 @@
-          :class="showTocDropdown ? 'transform rotate-90' : 'transform rotate-0'"
-        >
-          <icons-carat />
-        </div>
-      </button>
-      <div
-        :class="showTocDropdown === false ? 'hidden' : ''"
-        class="space-y-1 sm:space-y-2 mb-4 lg:mt-0 lg:block"
-      >
-        <p class="type-overline text-brand-accent hidden lg:block mt-0 mb-4 pl-6">
-          On this page
-        </p>
-        <ol
-          v-if="toc && toc.links"
-          class="list-none p-0"
+            <p
+              v-if="section"
+              class="type-overline text-brand-accent"
+            >
+              {{ section.title }}
+            </p>
+            <h1
+              v-if="title"
+              class="font-display text-3xl tracking-tight text-slate-900"
+            >
+              {{ title }}
+            </h1>
+          </header>
+          <Prose>
+            <slot />
+          </Prose>
+        </article>
+      </div>
+      <div class="hidden xl:sticky border-none xl:top-[4.5rem] xl:-mr-6 xl:block xl:h-[calc(100vh-4.5rem)] xl:flex-none xl:py-16 xl:pr-6">
+        <nav
+          aria-labelledby="on-this-page-title"
+          class="w-56"
         >
-          <div
-            v-for="heading in toc.links"
-            :key="heading.text"
+          <h2
+            v-if="toc && toc.links"
+            id="on-this-page-title"
+            class="type-overline text-brand-accent"
           >
-            <a
-              :href="`#${heading.id}`"
-              class="type-body-3 text-content-tertiary hover:underline hover:text-gray-900"
-              @click="handleTocClick(heading.id)"
+            On this page
+          </h2>
+          <ol class="mt-4 text-sm">
+            <li
+              v-for="contentSection in toc.links"
+              :key="contentSection.id"
             >
-              <li
-                class="m-0 py-3 px-6 hover:bg-gray-50"
-                :class="visibleHeadingId === heading.id ? 'bg-gray-100' : 'bg-transparent'"
+              <NuxtLink
+                :href="`#${contentSection.id}`"
+                @click="handleTocClick(contentSection.id)"
               >
-                {{ heading.text }}
-              </li>
-            </a>
-            <ul
-              v-if="heading.children && heading.children.length > 0"
-              class="pl-0 m-0"
-            >
-              <a
-                v-for="subHeading in heading.children"
-                :key="subHeading.text"
-                :href="`#${subHeading.id}`"
-                class="type-body-3 text-content-tertiary hover:underline hover:text-gray-900"
-                @click="handleTocClick(subHeading.id)"
+                <h3
+                  class="border-l p-squish-2 hover:text-content-primary hover:border-content-primary hover:border-l-2"
+                  :class="visibleHeadingId === contentSection.id ? 'text-content-primary border-brand-accent border-l-2': 'font-normal text-content-tertiary'"
```

Comment:
@tom-blake @AndyClifford I checked the design and it was correct that the active item in the table of contents has the brand accent colour. 
See: https://www.figma.com/file/IgRlQWJTyGoZhlF4fVKD6B?node-id=5:1138#343899036